tests/websocket_new_test.py

```diff
@@ -317,17 +316,17 @@ def test_accept_deflate_ext_window_max_bits_13(self):
             # The server responds the correct Websocket handshake
             # print('Extension offer: %r' % extension_string)
             match = re.match(self.handshake_re, result)
-            self.assertIsNotNone(match)
-            self.assertEqual(1, len(match.groups()))
+            assert match is not None
+            assert len(match.groups()) == 1
 
             offered_parts = [part.strip().lower() for part in extension_string.split(';')]
             offered_parts_names = [part.split('=')[0].strip() for part in offered_parts]
             offered_parts_dict = dict(zip(offered_parts_names[1:], vals))
 
             accepted_ext_parts = match.groups()[0].decode().split('; ')
-            self.assertEqual(accepted_ext_parts[0], 'permessage-deflate')
+            assert accepted_ext_parts[0] == 'permessage-deflate'
             for param, val in (part.split('=') for part in accepted_ext_parts[1:]):
-                self.assertLessThanEqual(int(val), offered_parts_dict[param])
+                assert int(val) == offered_parts_dict[param]
```

Comment:
`assertLessThanEqual -> assert <=`
are you sure strict equal is proper here?

Reply:
The specification says, it has to be less or equal, but our implementation is always using the offered value, thus avoiding possible conflicts. While I initially thought to use `<=`, I decided it provides no/very little additional value to allow the code to return a smaller value than specified in the negotiation offer and future revisions should attempt to use the same value as well.

Reply:
Thanks.